packages/EasyCore/src/Interfaces/DatabaseEntityInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Interfaces;
+
+interface DatabaseEntityInterface
```

Comment:
Does it mean that every project will have to make their entities implement this interface for the logic to work?

Why is that needed?

Reply:
We are filtering entities by this interface in the current implementation.
So, looks like we need it.

Reply:
Is it a requirement from doctrine for a class to implement a specific interface?
No, so why do we need an interface?

Reply:
We don't need it. Removed